LimerickStreetArt.Web/API/RootController.cs

```diff
@@ -0,0 +1,28 @@
+﻿using LimerickStreetArt.Web.Api;
+using Microsoft.AspNetCore.Mvc;
+
+namespace LimerickStreetArt.Web.API
+{
+	[Route("/")]
+	[ApiController]
+	public class RootController : Controller
```

Comment:
https://stackoverflow.com/questions/55239380/why-derive-from-controllerbase-vs-controller-for-asp-net-core-web-api

Don't create a web API controller by deriving from the Controller class. Controller derives from ControllerBase

Change `RootController : Controller`
to 
`RootController : ControllerBase`